### PR TITLE
Add App Extension model

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it adds AppExtension model

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3839 New model AppExtension](https://app.clickup.com/t/2549495/SALEOR-3839)

**Review by fowczarek — APPROVED** [review]

File: saleor/app/models.py [hunk 1]
```diff
@@ -103,6 +108,22 @@ class AppToken(models.Model):
     auth_token = models.CharField(default=generate_token, unique=True, max_length=30)
 
 
+class AppExtension(models.Model):
+    app = models.ForeignKey(App, on_delete=models.CASCADE, related_name="extensions")
+    label = models.CharField(max_length=256)
+    url = models.URLField()
+    view = models.CharField(choices=AppExtensionView.CHOICES, max_length=128)
+    type = models.CharField(choices=AppExtensionType.CHOICES, max_length=128)
+    mounting_place = models.CharField(
```
Comment (maarcingebala) [inline saleor/app/models.py:117]: I'm not convinced about this name, maybe `target` would be a good name?

Comment (korycins) [reply saleor/app/models.py:117]: Do you mean - replacing `mounting_place` to `target` or `target_place`?

Comment (maarcingebala) [reply saleor/app/models.py:117]: I mean just `target` and `AppExtensionTarget`.

**Pull request merged**